cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsRdsLaunchService.java

```diff
@@ -81,10 +81,11 @@
             cfRetryClient.describeStacks(new DescribeStacksRequest().withStackName(cFStackName));
             LOGGER.debug("Stack already exists: {}", cFStackName);
         } catch (AmazonServiceException ignored) {
-            boolean existingVPC = awsNetworkView.isExistingVPC();
+            // boolean existingVPC = awsNetworkView.isExistingVPC();
```

Comment:
What is the plan here?

Reply:
Not sure ... I'll think about it. RDS doesn't require you to state a VPC, but maybe it should be present for some other reason.

Reply:
I will address this in a follow-up change.